src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -186,7 +171,10 @@ export function getPsvTechRecord(dtpNumbersFromRefData: FormNodeOption<string>[]
         width: FormNodeWidth.XS,
         type: FormNodeTypes.CONTROL,
         editType: FormNodeEditTypes.NUMBER,
-        validators: [{ name: ValidatorNames.Max, args: 99 }, { name: ValidatorNames.Required }],
+        validators: [
```

Comment:
Does setting the edit type also not let you type numbers? Or do you need to add that as welll?

Reply:
the NUMBER edit type makes it so you can only input numbers